Sources/Purchasing/Offerings.swift

```diff
@@ -45,14 +45,32 @@ import Foundation
         return all[currentOfferingID]
     }
 
+    internal let response: OfferingsResponse
```

Comment:
Just curious, seems like we are storing the response to be able to store it in UserDefaults because `Offerings` isn't `Codeable` correct? I wonder if that's something we could add to avoid having to parse it again later? In any case, nothing change here.

Reply:
We can't make `Offerings` `Codable` because it has the full `StoreProduct`s inside, so we need to load those from the `OfferingsResponse` data.